volatility3/framework/layers/codecs/__init__.py

```diff
@@ -1,8 +1,320 @@
-# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
 # which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
-#
 
-"""Codecs used for encoding or decoding data should live here
 
+# Temporary comment:
```

Comment:
I wonder what's the best way to integrate algorithms into the "codecs" from the framework standpoint. 
What are your recommandations about that ? Should I implement algorithms In seperate files ? I feel like I should but I have put both algorithms here for now :)

Reply:
Thanks, I'll get right on that.